examples/contact-sensor-app/linux/include/CHIPProjectAppConfig.h

```diff
@@ -36,4 +36,4 @@
 
 #define CHIP_DEVICE_ENABLE_PORT_PARAMS 1
 
-#define CHIP_DEVICE_CONFIG_DEVICE_NAME "Test Bulb"
+#define CHIP_DEVICE_CONFIG_DEVICE_NAME "Contact sensor"
```

Comment:
In other places in the code I can see device names in title case, i.e. "Contact Sensor"

Reply:
I believe this is better than before even if case does not matter. We can fix consistency separately.